src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -125,6 +125,19 @@ interface Sections {
   next?: LinkedArchiveTreeSection | undefined;
 }
 
+export const getPageIndex = (
+  tree: ArchiveTree,
+  pageId: string
+): number | undefined => {
+  const pages = findTreePages(tree);
+
+  for (let index = 0; index < pages.length; index++) {
+    if (pages[index].id === pageId) {
+      return index;
+    }
+  }
```

Comment:
For fun I tried to think of a more succinct version, though I'm not saying it's better than the plain for loop:
```
const index = pages.findIndex((p) => p.id === pageId);
return index === -1 ? undefined : index;
```

Reply:
This is an instance where returning -1 from javascript index methods is super annoying lol

Reply:
Ah using the builtin findIndex is probably a good idea. I'll change.